Cargo.lock

```diff
@@ -492,9 +492,8 @@ checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
 
 [[package]]
 name = "chalk-derive"
-version = "0.75.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "d54e3b5f9e3425e6b119ff07568d8d006bfa5a8d6f78a9cbc3530b1e962e316c"
+version = "0.76.0-dev.0"
+source = "git+https://github.com/rust-lang/chalk?branch=master#f518c7c7ed723dd14486bc966e2474e46fe04269"
```

Comment:
This is temporary. Using this commit until a new chalk release that includes https://github.com/rust-lang/chalk/pull/740.